source/features/branch-buttons.tsx

```diff
@@ -12,7 +12,7 @@ import {getRepoURL, getOwnerAndRepo} from '../libs/utils';
 
 async function getTagLink() {
 	const tags = select
-		.all('.branch-select-menu .select-menu-list:last-child .select-menu-item')
+		.all<HTMLLIElement>('.branch-select-menu .select-menu-list:last-child .select-menu-item')
```

Comment:
Not sure if this is the right type. I guessed it's an LI based on the selector names.